helios-client/src/main/java/com/spotify/helios/client/HttpsHandlers.java

```diff
@@ -92,10 +122,85 @@ protected CertificateAndPrivateKey createCertificateAndPrivateKey() {
       final CertificateAndKeyPair certificateAndKeyPair =
           X509CertificateFactory.get(agentProxy, identity, getUser());
 
+      // try writing the generated cert & key to ~/.helios
+      try {
+        Files.createDirectories(HELIOS_HOME);
+
+        final String certData = X509CertificateFactory.CERT_ENCODING
+            .encode(certificateAndKeyPair.getCertificate().getEncoded());
+        final String certPem = String.format("%1$s\n%2$s\n%3$s",
```

Comment:
Can't we just use ordinary indexing here, ie `%s\n%s\n%s`? I actually didn't know about explicit indexing syntax, but it seems ordinary indexing is more readable.